website/docs/guides/migrate-from-spark-to-databricks.md

```diff
@@ -1,18 +1,34 @@
 ---
-title: "Migrating from dbt-spark to dbt-databricks"
-id: "migrating-from-spark-to-databricks"
+title: "Migrate from dbt-spark to dbt-databricks"
+id: "migrate-from-spark-to-databricks"
+description: Learn how to migrate from dbt-spark to dbt-databricks.
+displayText: Migrate from Spark to Databricks
+hoverSnippet: Learn how to migrate from dbt-spark to dbt-databricks.
+# time_to_complete: '30 minutes' commenting out until we test
+icon: 'guides'
+hide_table_of_contents: true
+tags: ['Migration', 'dbt Core','dbt Cloud']
+level: 'Intermediate'
+recently_updated: true
 ---
 
-You can [migrate your projects](#migrate-your-dbt-projects) from using the `dbt-spark` adapter to using the [dbt-databricks adapter](https://github.com/databricks/dbt-databricks). In collaboration with dbt Labs, Databricks built this adapter using dbt-spark as the foundation and added some critical improvements. With it, you get an easier set up &mdash; requiring only three inputs for authentication &mdash; and more features such as support for [Unity Catalog](https://www.databricks.com/product/unity-catalog).
+## Introduction
 
-## Simpler authentication
+You can migrate your projects from using the `dbt-spark` adapter to using the [dbt-databricks adapter](https://github.com/databricks/dbt-databricks). In collaboration with dbt Labs, Databricks built this adapter using dbt-spark as the foundation and added some critical improvements. With it, you get an easier set up &mdash; requiring only three inputs for authentication &mdash; and more features such as support for [Unity Catalog](https://www.databricks.com/product/unity-catalog).
+
+### Prerequisites
+
+- Your project must be compatible with dbt 1.0 or greater. Refer to [Upgrading to v1.0](/docs/dbt-versions/core-upgrade/upgrading-to-v1.0) for details. For the latest version of dbt, refer to [Upgrading to v1.3](/docs/dbt-versions/core-upgrade/upgrading-to-v1.3).
```

Comment:
Making a note to find a way to reference the `latest` version of dbt where it auto updates text like this. I'll have to check and see how often it occurs in our docs to see if that's worth it.
```suggestion
- Your project must be compatible with dbt 1.0 or greater. Refer to [Upgrading to v1.0](/docs/dbt-versions/core-upgrade/upgrading-to-v1.0) for details. For the latest version of dbt, refer to [Upgrading to v1.7](/docs/dbt-versions/core-upgrade/upgrading-to-v1.7).
```